packages/blade/src/components/Card/LinkOverlay.web.tsx

```diff
@@ -0,0 +1,29 @@
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components';
+import type { LinkOverlayProps } from './types';
+
+const LinkOverlay = styled.a<LinkOverlayProps>(
+  (): CSSObject => {
+    return {
+      // Need this to reset the button styles
+      all: 'unset',
+      appearance: 'none',
+      border: 0,
+      padding: 0,
+      position: 'static',
```

Comment:
not absolute?

Reply:
No has to be static: check https://www.sarasoueidan.com/blog/nested-links/

Added a comment to link this too